pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3259,17 +3260,17 @@ func (c *Cloud) getInstancesByNodeNamesCached(nodeNames sets.String) ([]*ec2.Ins
 	return instances, nil
 }
 
-func (c *Cloud) getInstancesByNodeNames(nodeNames []string) ([]*ec2.Instance, error) {
+func (c *Cloud) getInstancesByNodeNames(nodeNames []string, states ...string) ([]*ec2.Instance, error) {
 	names := aws.StringSlice(nodeNames)
```

Comment:
basically imo the function shoudl do what it says, getInstancesByNodeNames, not getRunningInstancesByNodeNames